cypress/e2e/buy-shirt.cy.ts

```diff
@@ -33,6 +34,7 @@ describe("Buy a t-shirt", () => {
     shippingStepPage.checkout();
 
     paymentStepPage.payByBank();
-    paymentStepPage.getConfirmationMessage();
+    const confirmationMessage = paymentStepPage.getConfirmationMessage();
+    confirmationMessage.should("have.text", expectedText);
```

Comment:
there is no need to create the const confirmationMessage
```suggestion
    paymentStepPage.getConfirmationMessage()
        .should("have.text", expectedText);
```